students/saleta_sebastian/lesson_08_regular_expressions/regex_version_of_strip.py

```diff
@@ -0,0 +1,19 @@
+import re
+
+
+def alternative_strip(word, character):
+    if not character:
+        word_regex = re.compile(r'\s+')
+        return word_regex.sub('', word)
+    else:
+        word_regex = re.compile(r'^[{}]+|[{}]+$'.format(character, character))
```

Comment:
It could be shorter: 
```word_regex = re.compile(r'^[{0}]+|[{0}]+$'.format(character))``` 
but this is just tip not needed to be fixed. 
You can cross check:
https://pyformat.info/